modules/social_features/social_event/social_event.module

```diff
@@ -1022,3 +1053,31 @@ function _social_event_get_enrollment_status(NodeInterface $event) {
 
   return $status;
 }
+
+/**
+ * Change views filters order.
+ *
+ * @param string $view_config
+ *   The views config name.
+ * @param string $display
+ *   The views display id.
+ */
+function _social_event_views_update($view_config, $display) {
```

Comment:
@dmytriikaiun Since this function is used only in `social_event.install` file I think it is better to move it there.

Reply:
@ribel It's also used in the `social_group.install`

Reply:
@dmytriikaiun Ok, then it's fine.